content/03.aim1.md

```diff
@@ -46,20 +48,79 @@ Simulation statistics under different parameters:
+| yan | 90 | 20214 | Embryo Devel | human | 6 | GSE36552 | Illumina HiSeq 2000 | celltypes derived from different embryonic developmental time points |
+| goolam | 124 | 41428 | Embryo Devel | mouse | 5 | E-MTAB-3321 | | 5 celltypes derived from different embryonic developmental time points |
+| pollen | 301 | 23730 | Developing Cerebral Cortex | human | 11 | SRP041736 | Illumina HiSeq 2000 | 11 celltypes defined by different celllines |
+| patel | 430 | 5948 | Glioblastoma | human | 12 | GSE57249 | Illumina HiSeq 2000 | 5 celltypes defined from five primary glioblastomas |
+| nestorowa | 1656 | 4773 | Glioblastoma | human | 12 | GSE81682 | Illumina HiSeq 2500 | 9 celltypes defined by sorting |
+| Petropoulos | 1529 | 26178 | Embryo Devel | human | 6 | E-MTAB-3929 | Illumina HiSeq 2000 | celltypes defined by developmental stages |
+| HCA melanoma data | 6639 | 22489 | Lymph node | mouse | 9 | HCA release | Smart-seq2 |
+| baron-human | 8569 | 20125 | Pancreas | human | 14 | GSE84133 | Illumina HiSeq 2500 | 14 celltypes defined based on expression profiles and cell origin |
+
+##### 2. Visualization of real single cell data based on differnt structure of VAE: VAE_depth3 (3 hidden layer), VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE, ZIFA, umap and PCA
+
+We tested the performance of VAE and another 4 methods (t-SNE, ZIFA, umap and PCA) on these real datasets with different number of cells, genes and celltypes. 
+Figure 4 shows the 2-D visulization of the real single cell datasets. 
+PCA and ZIFA generally performs better when number of cells is small, but with the increase of number of cells, VAE shows a good power to differentiate different celltypes.
+
+![](images/real.data.vis.png)
+
+Figure 4: 2D Visualization of real single cell data using VAE_depth3 (3 hidden layer), VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE, ZIFA, umap and PCA.
+
+##### 3. Performance evaluation
+
+Evaluation of model performance is based on three different types of metrics: clustering based, average silhouette score and knn-based.
+
+*Clustering based approach*: k-means clustering is performed based on the latent space, the model performance is measured by NMI and ARI. 
+
+![](images/true.celltype.clustering.png)
+
+Figure 5:  Performance comparison among VAE, t-SNE, ZIFA, umap and PCA based on k-means. For dataset with small number of cells, the performance of t-SNE, umap, ZIFA and PCA is better than VAE. VAE and umap outperforms t-SNE, ZIFA and PCA whith increase number of cells. 
+
+*Average silhouette score*: average silluetee score measures how well each points lies with its own cluster, which indicates the separability of each inidividual cluster. 
+The value of average silluetee score is between -1 to 1, with 1 means the cluster is far away from its neighboring clusters.
+
+<center>
+
+![](images/true.cell.type.average_sil_score.bmp)
```

Comment:
Is there a reason to make these into bitmaps? It's generally not a very efficient format for storing images.

Reply:
okay, I'll change the image type.